src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverMetrics.java

```diff
@@ -36,14 +36,16 @@
  *  Export metrics for the resource resolver bundle:
  *
  *  org.apache.sling.resourceresolver.numberOfResourcesWithAliasesOnStartup -- the total number of resources with sling:alias properties found on startup
+ *  org.apache.sling.resourceresolver.numberOfDetectedConflictingAliases -- the total number of detected invalid aliases
+ *  org.apache.sling.resourceresolver.numberOfDetectedInvalidAliases -- the total number of detected invalid aliases
+ *
  *  org.apache.sling.resourceresolver.numberOfResourcesWithVanityPathsOnStartup -- the total number of resources with sling:vanityPath properties found on startup
- *  org.apache.sling.resourceresolver.numberOfVanityPaths -- the total number of vanity paths in the cache
```

Comment:
This metric name is now missing from the comment, is that intended?

Reply:
It just moved down; I separated alias from vp metrics, and sorted them alphabetically